Sources/swift-doc/Supporting Types/Components/Relationships.swift

```diff
@@ -73,30 +73,36 @@ struct Relationships: Component {
     }
 
     var html: HypertextLiteral.HTML {
-        var graph = symbol.graph(in: module)
-        guard !graph.edges.isEmpty else { return "" }
-
-        graph.aspectRatio = 0.125
-        graph.center = true
-        graph.overlap = "compress"
+        guard sections.map(\.symbols.count).reduce(0, +) != 0 else { return "" }
 
-        let algorithm: LayoutAlgorithm = graph.nodes.count > 3 ? .neato : .dot
         var svg: HypertextLiteral.HTML?
+        var graph = symbol.graph(in: module)
+        if !graph.edges.isEmpty {
+            graph.aspectRatio = 0.125
+            graph.center = true
+            graph.overlap = "compress"
+
+            let algorithm: LayoutAlgorithm = graph.nodes.count > 3 ? .neato : .dot
 
-        do {
-            svg = try HypertextLiteral.HTML(String(data: graph.render(using: algorithm, to: .svg), encoding: .utf8) ?? "")
-        } catch {
-            logger.error("\(error)")
+            do {
+                svg = try HypertextLiteral.HTML(String(data: graph.render(using: algorithm, to: .svg), encoding: .utf8) ?? "")
+            } catch {
+                logger.error("\(error)")
+            }
```

Comment:
I think this has gotten complex enough to warrant extraction into a separate helper method / property (e.g. `var svg: HyperTextLiteral.HTML`. What do you think?

Reply:
Agree, moved to `graphHTML` property.